web-client/src/translations/translations.json

```diff
@@ -25,6 +25,15 @@
         "pleaseWaitLabel": "Please wait...",
         "tip_title": "Remember",
         "tip_info": "Once a volunteer offers to help, you must accept their help in order for them to see your contact information."
+      },
+      "newRequest": {
+        "title": "New Request",
+        "form": {
+          "title": "Title",
+          "body": "Enter details of what is needed",
+          "consent": "Allow Push Notifications. You can always change this in your Settings later.",
+          "submit": "Send my need"
+        }
```

Comment:
You still need to add translations to Portuguese

Reply:
![image](https://user-images.githubusercontent.com/35262512/78777500-8d88e800-7991-11ea-8b3f-9c569dd1e699.png)

Done!

Reply:
Fixed here - https://github.com/reach4help/reach4help/pull/247/commits/98799d1fdae6b1e59cd038d2f8192ccb3382b953